src/Plugin.php

```diff
@@ -86,6 +90,45 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes svg width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
+      return preg_replace('@((?:width|height)=.+?\s+)@', '', $matches[0]);
```

Comment:
1. The intention of `.+?\s+` seems to be `[^\s]+\s+`

1. The outer parenthesis are not necessary and should be removed, because you do not use the capturing group.

1. This will also remove the width and height of images for which it was set manually and explicitly (e.g., logos in an article text). Didn't you state that the lack of image dimensions can be determined by `width="1" height="1"`?